VMS/Pages/ActivityInfoPage/InfoPage.razor

```diff
@@ -0,0 +1,74 @@
+﻿<div class="InfoPage">
```

Comment:
Please rename all class names, class names won't be capitalized.

Reply:
done